src/OpenTelemetry/Resources/ResourceBuilder.cs

```diff
@@ -30,15 +30,22 @@ private ResourceBuilder()
         {
         }
 
+        private static Resource DefaultResource { get; } = new Resource(new Dictionary<string, object>
+        {
+            [ResourceSemanticConventions.AttributeServiceName] = "unknown_service"
+                + (string.IsNullOrWhiteSpace(System.Diagnostics.Process.GetCurrentProcess().ProcessName)
+                ? ":" + System.Diagnostics.Process.GetCurrentProcess().ProcessName : string.Empty),
+        });
+
         /// <summary>
-        /// Creates a <see cref="ResourceBuilder"/> instance with SDK defaults
-        /// added. See <a
-        /// href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/">resource
+        /// Creates a <see cref="ResourceBuilder"/> instance with Default
+        /// service.name added. See <a
+        /// href="https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service/">resource
```

Comment:
I'd link to this section: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#semantic-attributes-with-sdk-provided-default-value
instead.